Lib/fontTools/subset/__init__.py

```diff
@@ -2191,7 +2191,9 @@ def prune_post_subset(self, options):
                 subrs = getattr(c.private, "Subrs", [])
                 decompiler = _DehintingT2Decompiler(css, subrs, c.globalSubrs)
                 decompiler.execute(c)
+            pen = NullPen()
             for charstring in css:
+                charstring.draw(pen)  # this will set the charstring's width
```

Comment:
It doesn't, but will set the width on decompiler.  We can check and copy there and avoid an extra draw().  WDYT?